haeun/time-machine/src/main/java/com/meyame/timemachine/exception/CustomException.java

```diff
@@ -0,0 +1,21 @@
+package com.meyame.timemachine.exception;
+
+import com.meyame.timemachine.exception.code.CInterface;
+import lombok.Getter;
+
+@Getter
+public class CustomException extends RuntimeException{
```

Comment:
관례! [Java Style Guide](https://newwisdom.tistory.com/m/96)를 알아보면 앞으로 더 가치있는 개발자가 될 수 있을거야~
```suggestion
public class CustomException extends RuntimeException {
```

Reply:
네! 유용한 자료 감사합니다 :)